events/internal/test/helpers.go

```diff
@@ -0,0 +1,56 @@
+// internal test helpers for NATS kv stuff. Yes this means the tests are
+// strictly "integration tests." Many electrons have been spilt in the NATS
+// space as to whether they can/should/will provide something mockable for
+// unit testing with or without fault injection. The current answer is "look
+// at our tests for examples." While this is strictly speaking unsatisfying
+// this is what we have.
+
+//nolint:all
+package test
+
+import (
+	"os"
+	"testing"
+	"time"
+
+	"github.com/nats-io/nats-server/v2/server"
+	srvtest "github.com/nats-io/nats-server/v2/test"
+	"github.com/nats-io/nats.go"
+)
+
+// XXX: this will panic on an error
+func StartJetStreamServer(t *testing.T) *server.Server {
```

Comment:
cool! now I have no excuse to not add integrations tests for the other events methods that utilize the jetstream.